### DISCUSSION TITLE
[Social sharing] <sharing to facebook/ig etc?>

### DISCUSSION THREAD

**Discussion by blistovmhz** [immich-app/immich#5411] [Feature Request]
### The feature

I'm surprised I couldn't find this brought up anywhere? Maybe I'm tarding out? There seems to be no way to share to various social media platforms, which seems silly as it's such an integral part of our media.
Is this on the drawing board or has it even been requested?

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by ajnart** [comment]
If you click on the "Share" button on the mobile app it already downloads the picture and allows you to share it to other platforms is that what you are refering to ? That button does not exist on web using a desktop because you'd want to manually download the file and then go to that platform and share it there.

**Reply by blistovmhz** [reply]
I hate mobile :dagger:

**Reply by moccassins** [reply]
Hi, I miss this feature too. However, my use case is: You are in a social media app such as Instagram, WhatsApp, Telegram or similar and want to use a picture from immich. In WhatsApp or Telegram, you use the attachment button and usually select the gallery to search for and send the image. In Instagram, e.g. when creating a new story, a gallery selection opens automatically.

The problem with this is that Immich does not appear there and in Instagram you don't even have the option of selecting a different source.

The idea behind this is to save photos via Immich and only have small thumbnails for selection on the device. Once an image has been selected based on the thumbnail, the original must be downloaded from the server and forwarded to the social media app.

Unfortunately, I have not yet found a suitable solution for this use case anywhere, except for Google Photos.
I really hope that Immich will be able to do this at some point. Should I open a new feature request for this or can we include it here?

Greetings Mocca

**Reply by bo0tzz** [reply]
@Moccassins you're looking for #1641

**Comment by bo0tzz** [comment]
From a quick look at google photos, they have a few social media sharing buttons but those just generate a shared link and then post that. That's trivial enough that I don't see much value in adding it to Immich. Especially so since social media sharing buttons usually come with a bunch of tracker gunge and such.
